STYLEGUIDE.md

```diff
@@ -43,7 +43,9 @@ Assume the reader is a busy non-native English speaker.
       - Sourcegraph['s] Firefox add-on
       - Sourcegraph['s] Safari extension
 
-When specifically distinguishing between Server and Data Center, it might help to say "the [single-node] Server deployment option" and "the Data Center [cluster] deployment option".
+When referring to the build result of the open-source repository, use the name Sourcegraph OSS.
+
+When specifically distinguishing between Core and Sourcegraph OSS it is important to note that Core is not built from the open-source code base and Sourcegraph OSS does not include the ability to upgrade, or Sourcegraph extensions. These are included in the Core version of Sourcegraph, that is built from the same code as Sourcegraph Enterprise and has the ability to upgrade later.
```

Comment:
Technically Sourcegraph OSS supports Sourcegraph extensions, just not accessing the extension registry on Sourcegraph.com.

Reply:
How's this:

Reply:
Why that instead of just "...the ability to upgrade or access the extension registry on Sourcegraph.com"?

Reply:
I want to call out Sourcegraph extensions by name

Reply:
My version is more accurate. Referring to "public Sourcegraph extensions" implies that they can use private Sourcegraph extensions (which they can't, at least not in the same way that Sourcegraph Enterprise can). It also makes the mechanism unclear and sound less legitimate, as though we're disabling functionality instead of just limiting access to an online service.